src/components/LocationAutocomplete/LocationAutocomplete.scss

```diff
@@ -1,4 +1,4 @@
-$icon-margin-adjustment: 1px 1px 0px 3px;
+$icon-margin-adjustment: 7.5px 1px 0px 3px;
 
```

Comment:
The height of the input is 34px, but for the old version is 36px. Could you please align it?

Reply:
Updated Autosuggest to 36px

Reply:
The height of the input is changing while focusing/blurring the input:
https://github.com/textkernel/oneui/assets/19436304/41568cf8-a45d-44af-a1d5-e8ddcd130454

Reply:
:+1: